dotnet/Razorvine.Pyrolite/Pyrolite/Pyrolite.csproj

```diff
@@ -11,6 +11,9 @@
     <Version>4.24.0</Version>
     <AssemblyVersion>4.24.0.0</AssemblyVersion>
     <FileVersion>4.24.0.0</FileVersion>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>false</DelaySign>
```

Comment:
I believe the default is `false`, so there's no need to set this property explicitly.

Reply:
However, you may want to add:

```
<PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
```

I'm not sure if that is needed with the latest SDK version or not. But when building on non-Windows in earlier versions it would fail.

Reply:
Nevermind on that `PublicSign` comment. It is no longer needed with the latest SDK version. I didn't add it to Apache Arrow, and everything works fine over there:

https://github.com/apache/arrow/blob/f913d8f0adff71c288a10f6c1b0ad2d1ab3e9e32/csharp/Directory.Build.props#L18-L22